flash/image/embedding/vissl/adapter.py

```diff
@@ -126,6 +126,7 @@ def from_task(
     def get_model_config_template():
         cfg = AttrDict(
             {
+                "BASE_MODEL_NAME": "multi_input_output_model",
```

Comment:
Is it okay if it defaults to `multi_input_output_model`?

Reply:
Okay, looks like this is fine. `vissl` default config also includes this: https://github.com/facebookresearch/vissl/blob/main/vissl/config/defaults.yaml#L504.